chart/localdb/charts/dbdump/templates/dbdump-job.yaml

```diff
@@ -48,16 +48,11 @@ spec:
                 secretKeyRef:
                   name: compass-postgresql
                   key: postgresql-director-db-name
-            - name: DB_SSL
-              valueFrom:
-                secretKeyRef:
-                  name: compass-postgresql
-                  key: postgresql-sslMode
             command:
               - "/bin/bash"
             args:
               - "-c"
-              - "sleep 40; [ -f /tmp/dbdata/dump.sql ] && PGPASSWORD=${DB_PASSWORD} psql --username=${DB_USER} --host=${DB_HOST} --port=${DB_PORT} --dbname=${DB_NAME} --single-transaction -f /tmp/dbdata/dump.sql --set ON_ERROR_STOP=on --set sslmode=${DB_SSL} || echo 'There is no dump.sql to be applied.' ; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
+              - "sleep 40; [ -f /tmp/dbdata/dump-local ] && PGPASSWORD=${DB_PASSWORD} pg_restore -Fd -j 8 -O -x -U ${DB_USER} -h ${DB_HOST} -p ${DB_PORT} -d ${DB_NAME} tmp/dump-local || echo 'There is no dump.sql to be applied.' ; exit_code=$?; echo '# KILLING PILOT-AGENT #'; pkill -INT cloud_sql_proxy; curl -XPOST http://127.0.0.1:15020/quitquitquit; sleep 5; exit $exit_code;"
```

Comment:
why `dump-local`? It looks like in the installation/run.sh script the dump folder is just `dump`